virt/virtual_machines/vm_networking/virt-using-the-default-pod-network-with-virt.adoc

```diff
@@ -30,3 +30,8 @@ include::modules/virt-networking-wizard-fields-web.adoc[leveloffset=+2]
 include::modules/virt-template-vm-config.adoc[leveloffset=+2]
 
 include::modules/virt-template-windows-vmi.adoc[leveloffset=+2]
+
+include::modules/virt-creating-a-service-from-a-virtual-machine.adoc[leveloffset=+1]
+
+== Additional resources
```

Comment:
This should have an anchor/ID added to it. Also since this is within a module, it should be `.Additional resources`, as using `== Additional resource` should be saved for the assembly level (see the Note box in [Writing assemblies](https://github.com/bergerhoffer/openshift-docs/blob/updating-style-guidelines-terms/contributing_to_docs/doc_guidelines.adoc#writing-assemblies)).

Reply:
In assembly, changed to:

include::modules/virt-creating-a-service-from-a-virtual-machine.adoc[leveloffset=+1]
.Additional resources
* xref:../../../networking/configuring_ingress_cluster_traffic/configuring-externalip.adoc#configuring-externalip[Configuring external IPs]